src/coreclr/jit/hwintrinsic.h

```diff
@@ -205,7 +205,10 @@ enum HWIntrinsicFlag : unsigned int
     HW_Flag_EmbBroadcastCompatible = 0x8000000,
 
     // The intrinsic is an embedded rounding compatible intrinsic
-    HW_Flag_EmbRoundingCompatible = 0x10000000
+    HW_Flag_EmbRoundingCompatible = 0x10000000,
+
+    // The intrinsic is an embedded masking incompatible intrinsic
+    HW_Flag_EmbMaskingIncompatible = 0x20000000,
```

Comment:
likewise this? should it be under `#ifdef FEATURE_HW_INTRINSICS`?

Reply:
This is in the hwintrinsics header, so its already effectively `FEATURE_HW_INTRINSICS`